backend/qemu.pm

```diff
@@ -897,7 +897,7 @@ sub start_qemu ($self) {
             else {
                 die "unknown NICTYPE $vars->{NICTYPE}\n";
             }
-            my $bootorder = $vars->{PXEBOOT} ? "bootindex=$i" : '';
+            my $bootorder = $vars->{PXEBOOT} ? "bootindex=@{[$i+1]}" : '';
```

Comment:
Does this syntax create an array-ref with one element and then just de-references that to be able to use interpolation? Maybe it would make sense to use a simpler syntax:

```suggestion
            my $bootorder = $vars->{PXEBOOT} ? 'bootindex=' . ($i + 1) : '';
```